gcovr/tests/test_args.py

```diff
@@ -160,6 +160,10 @@ def test_non_existing_directory_html_nested(capsys):
     helper_test_non_existing_directory_output(capsys, "--html-nested")
 
 
+def test_non_existing_directory_html_template_dir(capsys):
+    helper_test_non_existing_directory_output(capsys, "--html-template-dir")
```

Comment:
The naming doesn't fit here since this isn't an output option.

Reply:
Ah yeah and it wasn't even really testing something useful.  I've updated it.